lib/registration.pm

```diff
@@ -553,12 +553,14 @@ sub fill_in_reg_server {
     }
     else {
         send_key "alt-i";
-        if (sle_version_at_least('15')) {
+        if (sle_version_at_least('12-SP3')) {
```

Comment:
Is it possible to use the "new" version of `is_sle` for testing the version number?
In this case it could be `if (is_sle '12-sp3+') {`.

Reply:
sle_version_at_least is now deprecated, so we prevent new usages and once old code is refactored, we also get rid of it. Thanks!